### ISSUE TITLE
X-ticks displayed outside of actual chart area during zoom

### ISSUE THREAD

**Issue by florathenb** [masayuki0812/c3#1632]
you can see the behaviour in the zoom example. http://c3js.org/samples/interaction_zoom.html

When you zoom in a little bit and then drag the series around, you can see, that the labels appear already on the left side of the y-axis and not, how expected, directly beneath the y-axis.

**Comment by aendrew** [comment]
Yeah, that looks buggy. There should be a clipping mask there or something...
